src/acknowledgements/README.md

```diff
@@ -0,0 +1,11 @@
+# Acknowledgements
+
+We gratefully acknowledge feedback and support from our colleagues including
+John Baldwin, Kevin Brodsky, Brooks Davis, and Laurence Tratt.
+
+This work was supported by Innovate UK project "Digital Security by Design
+(DSbD) Technology Platform Prototype", 105694.
+We gratefully acknowledge UK Research and Innovation (UKRI), who sponsored the
+creation of Morello, and also the significant investment by DARPA in
+supporting the creation of CHERI and its earlier prototypes.
+We also acknowledge Arm Limited and Google, Inc.
```

Comment:
Is it intentional that there are duplicates in the front matter and here?

Reply:
(for the TR don't we normally put such sponsorship on page 2 and thus don't need it here either?)

Reply:
It is intentional, but it’s a reasonable question as to whether it’s a good idea. Certainly we need the sponsor acknowledgements (and disclaimers if appropriate) prominently displayed in the cover matter. But our approach in other TRs has varied -- paper-derived ones tend to also include an Acks section with sponsors, whereas “written to be a TR” ones don’t, I think?